features/shipping/applying_shipping_method_rules/viewing_available_shipping_methods_based_on_channel.feature

```diff
@@ -0,0 +1,51 @@
+@applying_shipping_method_rules
```

Comment:
Probably it would be better to introduce new namespace for these features
```suggestion
@viewing_shipping_methods
```

Reply:
you mean changing catalog's name also? There are other features starting with `viewing_shipping_methods`

Reply:
![Zrzut ekranu 2022-02-10 o 16 54 35](https://user-images.githubusercontent.com/53942444/153446515-9db054d9-f749-4dd9-8311-0f62241a2063.png)